lib/test_track.rb

```diff
@@ -57,4 +61,8 @@ def private_url
   def enabled?
     enabled_override.nil? ? !Rails.env.test? : enabled_override
   end
+
+  def feature_enabled?(split_name, context:)
+    app.test_track_ab(split_name, context: context)
```

Comment:
There was some discussion of dropping the context here, do we want to do that? Or just default it to 'global' or something?

Reply:
I opened a separate pr here https://github.com/Betterment/test_track_rails_client/pull/61, but I/we need to do some more research to see if anyone actually uses context, and then figure out if we should just remove it altogether. I didn't want this to be blocked on figuring that out

Reply:
What about auto-suffixing with _enabled or something? It’s gonna be weird to do `feature_enabled?(:thing_enabled)` and _enabled is now officially a meaningful suffix so it’s not crazy.

But on the other hand I’m not sure I like making the identity - app in this case - implicit. Is this sugar method even necessary or should you just use ab directly?

Reply:
What about `TestTrack.app.feature_enabled?(:thing)` where the split is called thing_enabled

Reply:
That’d presumably be a feature of identity, which I think is appropriate because we basically often use vary for aab split tests and ab for feature gates, and ab is a weird syntax to use for gating.

Reply:
i just wanted to build easy access to the one use case that we know about, which is a simple question of whether or not a feature is enabled (as opposed to exposing the entire `TestTrack::Identity` api)

I like the idea of auto-prefixing, but that also might make it slightly harder to find all usages of a feature flag (e.g., i usually search for all usages of `my_new_feature_enabled` when cleaning up an old split, if the name is different between the app user and a normal user, it might be harder to find).

Are you suggesting adding another method to TestTrack::Identity called `test_track_feature_enabled?` that is mostly an alias of `ab` but with some prefixing?

Reply:
Yeah I guess that’s pretty verbose too. And I hear you re greppability. Riffing - `TestTrack.app_ab(:thing_enabled)` or something like that reduces the number of new concepts right now?

Reply:
I’d love to find a way to get comfortable with TestTrack.feature_enabled?(...)

But if that doesn’t work then I’d vote for TestTrack.ab(...) over app_ab because the app part feels implied at this scope.

Reply:
Alternatively, I’d take 

`TestTrack.app.ab(:thing_enabled)`

That feels good to me. It makes it clear that we’re not magically loading in a visitor somewhere but instead we're working with an "app visitor." Given that, I think we should make `TestTrack.app` act the same as a VisitorDSL

Reply:
We spoke in-person. We're going to do `TestTrack.app_ap(:thing_enabled)` for now. We feel like that allows us to hide the parts of the `ApplicationIdentity` that don't make sense at the scope of an application, like `vary` which only makes sense for split-tests (a thing that doesn't make sense at the application level, we think).